package.json

```diff
@@ -235,6 +235,8 @@
 	"bugs": "https://github.com/preactjs/preact/issues",
 	"homepage": "https://preactjs.com",
 	"devDependencies": {
+		"@actions/github": "^5.0.0",
+		"@actions/glob": "^0.2.0",
```

Comment:
These are only used for typings. Happy to remove them if people prefer to not have the additional devDeps

Reply:
all good.